provisioner/clusterpy_test.go

```diff
@@ -533,7 +533,7 @@ func TestDecommissionDoesNotSupportProvider(t *testing.T) {
 		Provider: "zalando-eks",
 	}
 
-	p := clusterpyProvisioner{}
+	p := ZalandoEKSProvisioner{}
```

Comment:
Why is the expected result `ErrProviderNotSupported`? Shouldn't the `ZalandoEKSProvisioner` support the cluster when `cluster.Provider` is `zalando-eks`?